web/app/scripts/src/services/fields-config.es.js

```diff
@@ -6,6 +6,7 @@ export const moduleProperties = {
 };
 
 const module = angular.module(moduleProperties.moduleName, [
+    'incuna-surveys-fields.templates',
```

Comment:
Yes I know that you have this problem in testing, but in the project these templates should be compiled by the project into a global templates module. That was the idea I believe, however not sure how you can mock it for the tests. Maybe you can load this module only in the tests?